--- origsrc/gnome-system-monitor-3.0.1/src/Makefile.am	2011-03-24 19:39:45.000000000 -0500
+++ src/gnome-system-monitor-3.0.1/src/Makefile.am	2011-05-25 01:25:26.780939600 -0500
@@ -36,7 +36,7 @@ gnome_system_monitor_SOURCES = \
         gsm_color_button.c gsm_color_button.h
 
 
-gnome_system_monitor_LDADD = @PROCMAN_LIBS@ libbacon.la
+gnome_system_monitor_LDADD = libbacon.la @PROCMAN_LIBS@
 
 noinst_LTLIBRARIES = libbacon.la
 libbacon_la_SOURCES = \
